api/envoy/config/filter/http/adaptive_concurrency/v2alpha/adaptive_concurrency.proto

```diff
@@ -68,4 +69,8 @@ message AdaptiveConcurrency {
     GradientControllerConfig gradient_controller_config = 1
         [(validate.rules).message.required = true];
   }
+
+  // If set to false, the adaptive concurrency filter will operate as a pass-through filter. If a
```

Comment:
I think the defaults should be captured in the docs for RuntimeFeatureFlag. Can you clean this up?